specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -2620,6 +2623,13 @@
           "format": "date-time",
           "description": "Timestamp of last rotation of the Key Vault Key.",
           "x-ms-client-name": "LastKeyRotationTimestamp"
+        },
+        "currentVersionedKeyExpirationTimestamp": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "Timestamp of last rotation of the Key Vault Key.",
+          "x-ms-client-name": "This is a read only property that represents the expiration time of the current version of the customer managed key used for encryption."
```

Comment:
"x-ms-client-name" is used for define the property name in client tools like SDK.
This  "x-ms-client-name" is not correct. it looks more like description of the property. With this  "x-ms-client-name", the generated property name in SDK is very odd: "Thisisareadonlypropertythatrepresentstheexpirationtimeofthecurrentversionofthecustomermanagedkeyusedforencryption"

See generated SDK code: https://github.com/blueww/azure-sdk-for-net/blob/df2fb787318153c224396cf983191bbaa9c7a528/sdk/storage/Microsoft.Azure.Management.Storage/src/Generated/Models/KeyVaultProperties.cs#L49

Please remove "x-ms-client-name" (and add related info to description.), or use a short name for "x-ms-client-name".

Reply:
@HimanshuChhabra 
This issue blocks SDK generate good code. Would you please help to fix it?

Reply:
Yes, it was mistyped. Fixed it